docs/production/requirements.md

```diff
@@ -193,8 +193,7 @@ installing Zulip with a dedicated database server.
   subscribed (like on chat.zulip.org), add 20GB per (1000 user
   accounts) per (1M messages to public streams).
 
-- **Example:** When the
-  [chat.zulip.org](https://zulip.com/developer-community/) community server
+- **Example:** When [the Zulip development community](https://zulip.com/developer-community/) server
```

Comment:
Line needs to be wrapped to match the surrounding text, similar to the way it was structured before. :)